charts/urban-os/values.yaml

```diff
@@ -255,3 +255,7 @@ valkyrie:
   fullnameOverride: valkyrie
   enabled: true
   replicaCount: 1
+
+performancetesting:
+  url.ANDI_URL: ""
```

Comment:
can we just call this `ANDI_URL` instead of `url.ANDI_URL`?

Reply:
UrbanOS Values here has
```yaml
# urbanos toplevel
performancetesting:
  url.ANDI_URL
```

performancetesting chart has
```yaml
# performancetesting top level
url:
  ANDI_URL:
```

openshift deploy has
```yaml
# urbanos toplevel
performancetesting:
  ANDI_URL:
```

These three values need to be aligned.